### DISCUSSION TITLE
--base-path option is available in latest dockerhub image?

### DISCUSSION THREAD

**Discussion by codeMonkey-shin** [coder/code-server#6937] [General]
i use the option like this :
code-server --bind-addr 0.0.0.0:$SERVER_PORT --auth password --password yourpassword --base-path /proxy/$SERVER_PORT

then i had this error :
 error Unknown option --base-path
 
 I use this option to run my React app inside a code server and test it with a proxy URL.
 Is my method correct?

**Comment by code-asher** [comment]
`--base-path` is not a valid option unfortunately.  There is an issue to add it: https://github.com/coder/code-server/issues/6770

**Reply by codeMonkey-shin** [reply]
So, is there currently no way to test the web app through the code server?

**Reply by code-asher** [reply]
A few ways I think you can make it work:

1. Use the subdomain proxy instead, since it has no path issues.
2. Use relative paths in your app.
3. Normally code-server takes you to `/proxy/$port`, and it will rewrite the `/proxy/$port` path out of the request.  Instead, go to `/absproxy/$port`, which code-server does not rewrite, then in your React app you can configure the base path to be `/absproxy/$port` and it should work.

But, if you have to use the path proxy, and you have a base path *for code-server itself*, for example say you host code-server at `domain.com/code-server` (which means the full path is `/code-server/proxy/$port`), then this is not possible yet (this is what the `--base-path` flag will be for).
